NEXT_CHANGELOG.md

```diff
@@ -27,28 +27,30 @@ By [@USERNAME](https://github.com/USERNAME) in https://github.com/apollographql/
 
 ## ❗ BREAKING ❗
 
-### Configuration: Update metrics and healthcheck web endpoints, and make them configurable ([#1500](https://github.com/apollographql/router/issues/1500))
+### Configuration overhaul ([#1500](https://github.com/apollographql/router/issues/1500))
```

Comment:
I was trying to tweak some of the language in the existing `NEXT_CHANGELOG.md` about the health check and this just seemed like the better title for this, given the inclusion of this PR.